src-rs/build.rs

```diff
@@ -76,8 +76,8 @@ pub fn link_swift_package(package_name: &str, package_root: &str) {
     let swift_target_info = get_swift_target_info();
 
     println!(
-        "cargo:rustc-link-search=native={}.build/{}/{}",
+        "cargo:rustc-link-searchx={}.build/{}/{}",
```

Comment:
Is `searchx` a typo? Linking failed until I removed the `x`.
Also, I'm not sure that `native` should be removed. Doing so tells Cargo to search for dependencies and frameworks in addition to libraries, which isn't a use case I want to worry about currently.

Reply:
So for some reason the x was there and working with mine, replaced it with search=native and it is working fine with that.